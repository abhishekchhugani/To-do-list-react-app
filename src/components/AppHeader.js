//Component to display the header of the application

import React from 'react';
const AppHeader = () => {
  return (
    <nav>
      <div className="nav-wrapper blue">
        <div className="brand-logo center">
        <i className="medium material-icons">assignment</i>
          Todoer
        </div>
      </div>
    </nav>
  )
}

export default AppHeader;
