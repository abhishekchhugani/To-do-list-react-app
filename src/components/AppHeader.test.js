import React from 'react';
import ReactDOM from 'react-dom';
import { shallow } from 'enzyme';
import AppHeader from './AppHeader';

it('renders without crashing', () => {
    shallow(<AppHeader />);
});

it('snapshot test', () => {
    const wrapper = shallow(<AppHeader />);
    expect(wrapper).toMatchSnapshot();
});


it('renders without crashing', () => {
    const wrapper = shallow(<AppHeader />);
    wrapper.find(<i></i>).exists()
});
