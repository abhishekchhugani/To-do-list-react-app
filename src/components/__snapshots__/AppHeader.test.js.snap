// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot test 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <AppHeader />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "checkPropTypes": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <div
        className="nav-wrapper blue"
>
        <div
                className="brand-logo center"
        >
                <i
                        className="medium material-icons"
                >
                        assignment
                </i>
                Todoer
        </div>
</div>,
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <div
          className="brand-logo center"
>
          <i
                    className="medium material-icons"
          >
                    assignment
          </i>
          Todoer
</div>,
        "className": "nav-wrapper blue",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <i
              className="medium material-icons"
>
              assignment
</i>,
            "Todoer",
          ],
          "className": "brand-logo center",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "assignment",
              "className": "medium material-icons",
            },
            "ref": null,
            "rendered": "assignment",
            "type": "i",
          },
          "Todoer",
        ],
        "type": "div",
      },
      "type": "div",
    },
    "type": "nav",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <div
          className="nav-wrapper blue"
>
          <div
                    className="brand-logo center"
          >
                    <i
                              className="medium material-icons"
                    >
                              assignment
                    </i>
                    Todoer
          </div>
</div>,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <div
            className="brand-logo center"
>
            <i
                        className="medium material-icons"
            >
                        assignment
            </i>
            Todoer
</div>,
          "className": "nav-wrapper blue",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <i
                className="medium material-icons"
>
                assignment
</i>,
              "Todoer",
            ],
            "className": "brand-logo center",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "assignment",
                "className": "medium material-icons",
              },
              "ref": null,
              "rendered": "assignment",
              "type": "i",
            },
            "Todoer",
          ],
          "type": "div",
        },
        "type": "div",
      },
      "type": "nav",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getChildContext": Object {
            "calledByRenderer": false,
          },
          "getDerivedStateFromProps": Object {
            "hasShouldComponentUpdateBug": false,
          },
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
